packages/vue/src/components/molecules/SfGallery/SfGallery.vue

```diff
@@ -123,6 +132,42 @@ export default {
       type: Boolean,
       default: false,
     },
+    imageTag: {
```

Comment:
question: @AdamPawlinski @justyna-13  Let's discuss the approach on props here. Nuxt-image require a lot of props, so I'm thinking if we shouldn't put in one prop f.e nuxtImgConfig or something similar? Also we will need to update docs for this components.

Reply:
@aniamusial yes, I thought about such solution as well, but if we think about DX - it is easier to just use props mentioned in nuxt-image docs directly in SFUI components. Anyway it is worth discussing. Regarding docs I wait for the feedback from VSF to prepare docs for final version of this solution.

Reply:
after long thinking, I think Ania is right. I know it can be a bit worse DX at the beginning (looking at what props are needed) but for me, it has more pros. There are so many new props and not related to regular img tag so that it can also be confusing -  so it really depends which approach is worse for DX. Moreover, if anything changes in nuxt-image we don't have to change it at our side (I mean adding new props, removing or changing names). And when it comes to the problem that Adam said I think a solution here is to create great docs where all possible props will be described and links to related topic in nuxt-image